Quantum-Control-Applications/Superconducting/Two-Flux-Tunable-Transmons/Use Case 2 - Two-Qubit-Randomized-Benchmarking/two_qubit_rb_example.py

```diff
@@ -55,9 +59,10 @@ def bake_phased_xz(baker: Baking, q, x, z, a):
 
 # defines the CZ gate that realizes the mapping |00> -> |00>, |01> -> |01>, |10> -> |10>, |11> -> -|11>
 def bake_cz(baker: Baking, q1, q2):
-    q1_xy_element = f"q{q1}_xy"  #
-    q2_xy_element = f"q{q2}_xy"
-    q1_z_element = f"q{q1}_z"
+    q1_xy_element = f"q{q1_idx_str}_xy"
+    q2_xy_element = f"q{q2_idx_str}_xy"
+    q1_z_element = f"q{q1_idx_str}_z"
```

Comment:
The inputs `q1` and `q2` should now be `q1_idx_str`...

Reply:
When any of the 2Q gate bake functions get called, they are input `q1` and `q2`. The values these can take on are `1` and `2` or `2` and `1`. 

But, since CZ is a symmetric gate, it doesn't actually matter what is provided through these inputs, and so they are currently unused variables. Because of the way the code was written, however, they still need to be passed in as input, otherwise the bakery function will receive too many inputs. This is fixable in the future.

What IS however important to this function, is which elements will be called to implement the CZ gate, e.g. whether its qubits 3 and 5 or 4 and 1. This information is contained within the `q1_idx_str` and `q2_idx_str` and needs to be in this function.